client/fingerprint/bridge_linux.go

```diff
@@ -56,6 +56,14 @@ func (f *BridgeFingerprint) detect(module string) error {
 	// accumulate errors from every place we might find the module
 	var errs error
 
+	// Check if the module is in /sys/modules
+	loadedOrBuiltinPath := fmt.Sprintf("/sys/module/%s", module)
```

Comment:
Naming is always hard, so feel free to ignore this, but `loadedOrBuiltinPath` doesn't convey much to me 🤔 

I'm also not a Linux expert, so that very well may be due to my lack of knowledge 😅 

Skimming over [this paper](https://mirrors.edge.kernel.org/pub/linux/kernel/people/mochel/doc/papers/ols-2005/mochel.pdf) it seems that `/sys` is part of a broader feature set called  `sysfs`, so maybe this variable could be called something like `sysfsModulePath`?